Dockerfile

```diff
@@ -1,13 +1,20 @@
 FROM python:2.7.14-stretch
-ARG tensorflow_version=1.4.0
-ARG apache_beam_version=2.2.0
-WORKDIR /srv/sciencebeam
+ENV PROJECT_HOME=/srv/sciencebeam
+
+WORKDIR ${PROJECT_HOME}
 RUN virtualenv venv
-RUN venv/bin/pip install tensorflow==${tensorflow_version}
-RUN venv/bin/pip install apache-beam==${apache_beam_version}
+
+COPY requirements.prereq.txt ${PROJECT_HOME}/
+RUN venv/bin/pip install -r requirements.prereq.txt
+
 RUN venv/bin/pip install https://github.com/elifesciences/sciencebeam-gym/archive/develop.zip
```

Comment:
no versioning on this?

Reply:
not yet, but I was planning to use the commit like in _sciencebeam-texture_

(as I am not sure whether the separate _sciencebeam-gym_ project works well)